asserts/signtool/sign.go

```diff
@@ -122,6 +127,11 @@ func Sign(opts *Options, keypairMgr asserts.KeypairManager) ([]byte, error) {
 		}
 	}
 
+	if opts.UpdateTimestamp {
+		// Update the "timestamp" field with the current time in RFC3339 format.
+		headers["timestamp"] = time.Now().Format(time.RFC3339)
```

Comment:
could you add .UTC() ? it seems that is what we usually do when creating timestamps for assertions

Reply:
Just added the .UTC() and force-pushed a new commit, please review again when you have time, thanks!